src/test/scala/uk/gov/nationalarchives/notifications/LambdaSpecUtils.scala

```diff
@@ -82,4 +93,44 @@ class LambdaSpecUtils extends AnyFlatSpec with Matchers with BeforeAndAfterAll w
 
     super.afterAll()
   }
+
+  case class QueueHelper(queueUrl: String) {
+    val sqsClient: SqsClient = SqsClient.builder()
+      .region(Region.EU_WEST_2)
+      .endpointOverride(URI.create("http://localhost:8002"))
+      .build()
+
+    def send(body: String): SendMessageResponse = sqsClient.sendMessage(SendMessageRequest
+      .builder.messageBody(body).queueUrl(queueUrl).build())
+
+    def receive: List[Message] = sqsClient.receiveMessage(ReceiveMessageRequest
+      .builder
+      .maxNumberOfMessages(10)
+      .queueUrl(queueUrl)
+      .build).messages.asScala.toList
+
+    val visibilityTimeoutAttributes = new util.HashMap[QueueAttributeName, String]()
+    visibilityTimeoutAttributes.put(QueueAttributeName.VISIBILITY_TIMEOUT, (12 * 60 * 60).toString)
+
+    def createQueue: CreateQueueResponse = sqsClient.createQueue(
+      CreateQueueRequest.builder.queueName(queueUrl.split("/")(4)).attributes(visibilityTimeoutAttributes).build()
+    )
+    def deleteQueue(): DeleteQueueResponse = sqsClient.deleteQueue(DeleteQueueRequest.builder.queueUrl(queueUrl).build())
+
+    def queueAttribute(attributeName: QueueAttributeName): String = sqsClient.getQueueAttributes(
+      GetQueueAttributesRequest.builder().queueUrl(queueUrl).attributeNames(attributeName).build()
+    ).attributes().get(attributeName)
+
+    def nonVisibleMessageCount: Int = queueAttribute(QueueAttributeName.APPROXIMATE_NUMBER_OF_MESSAGES_NOT_VISIBLE).toInt
```

Comment:
This method isn't used anyywhere and neither are `queueAttribute` and `send` as far as I can tell. Will you need them later or is it safe to delete them?

Reply:
Deleted. That's what you get when you cut and paste stuff...